src/shape/bounding-box.js

```diff
@@ -200,29 +226,44 @@ Object.assign(pc, function () {
             return this._fastIntersectsRay(ray);
         },
 
+        /**
+         * @function
+         * @name pc.BoundingBox#setMinMax
+         * @description Sets the minimum and maximum corner of the AABB.
+         * Using this function is faster than assigning min and max separately.
+         * @param {pc.Vec3|number[]} min - The minimum corner of the AABB. Can be a {@link pc.Vec3} or an array (of length 3).
+         * @param {pc.Vec3|number[]} max - The maximum corner of the AABB. Can be a {@link pc.Vec3} or an array (of length 3).
+         */
         setMinMax: function (min, max) {
-            this.center.add2(max, min).scale(0.5);
-            this.halfExtents.sub2(max, min).scale(0.5);
+            if (!Array.isArray(min)) this._min.copy(min);
+            else this._min.set(min[0], min[1], min[2]);
+            if (!Array.isArray(max)) this._max.copy(max);
+            else this._max.set(max[0], max[1], max[2]);
```

Comment:
As @Maksims suggests, please add braces around `if` and `else` statements for readability.

Reply:
Done